### PR TITLE
Edit hide-inaccessible-columns docs section

### PR DESCRIPTION
**Pull request by jhlodin**
Minor writing and style edits

### CODE REVIEW

**Review by mosabua — APPROVED** [review]

File: docs/src/main/sphinx/security/overview.rst [hunk 1]
```diff
@@ -124,12 +124,13 @@ In addition, Trino :doc:`provides an API </develop/system-access-control>` that
 allows you to create a custom access control method, or to extend an existing
 one.
 
-The access control can limit access to columns of a table. The default behavior
-of an unqualified access to all columns with a ``SELECT *`` statement is to
+Access control can limit access to columns of a table. The default behavior
+of an unqualified query to all columns with a ``SELECT *`` statement is to
```
Comment (hashhar) [inline docs/src/main/sphinx/security/overview.rst:128]: too wordy. Just mention `SELECT *`. "unqualified query" isn't an actual concept.

**Review by hashhar — APPROVED** [review]
